### PR TITLE
Show database version in footer

### LINKED ISSUE #1156: FEAT: show database version in the footer
**Is your feature request related to a problem? Please describe.**
It's useful to get some additional information about the connected database.

**Describe the solution you'd like**
Show additional information like the database version at the footer bar
Example:
image

**Describe alternatives you've considered**

**Database type, if appropriate**
MySQL but if possible it should work for all supported databases.

### PR DESCRIPTION
**Pull request by niklas-stegmann**
I added a way to display the database version.
There is not much space in the footer, so I added a tooltip for the connection type where the version is displayed.

Maybe someone has a better idea?

### Demo
postgres

mariadb

sqlserver

sqlite

Fixes #1156

### CODE REVIEW

**Comment by Zerotask** [comment]
Would be great to see a summary directly in the footer, e. g. PostgreSQL 14.8

**Comment by rathboma** [comment]
I think this looks great. I'd actually like to utilize the window titlebar more for this type of information, although that doesn't really work on Mac, one step at a time. :-)

File: apps/studio/src/lib/db/clients/postgresql.ts [hunk 1]
```diff
@@ -1502,6 +1503,10 @@ async function runWithConnection<T>(x: Conn, run: (p: PoolClient) => Promise<T>)
   }
 }
 
+function getVersionString(version: VersionInfo): string {
+  return version.version.split(" on ")[0];
```
Comment (rathboma) [inline apps/studio/src/lib/db/clients/postgresql.ts:1507]: Does this work properly for Cockroach DB and Redshift do you think? Maybe it should return the full version for those?

File: apps/studio/src/lib/db/clients/sqlserver.js [hunk 2]
```diff
@@ -37,8 +37,11 @@ export default async function (server, database) {
   // light solution to test connection with with the server
   await driverExecuteQuery(conn, { query: 'SELECT 1' });
 
+  const version = await getVersion(conn);
```
Comment (rathboma) [inline apps/studio/src/lib/db/clients/sqlserver.js:40]: You can probably delete the previous line (SELECT 1) as getting the version will test the connection also

**Comment by rathboma** [comment]
@niklas-stegmann Question - can you double check cockroachdb & redshift versions will split properly given your `getVersion` function? I think the version strings are listed online in a bunch of places. Maybe add a comment with examples from each.

**Comment by niklas-stegmann** [comment]
@rathboma here are the results:

**CockroachDB:**

full version string:
image

displayed version:
image


**AWS Redshift**

full version string:
image

displayed version:
image

**Pull request merged**